.env

```diff
@@ -1,2 +1,3 @@
 TAG=latest
-LAKEFS_BLOCKSTORE_TYPE=local
\ No newline at end of file
+LAKEFS_BLOCKSTORE_TYPE=local
+DOCKER_REG=https://hub.docker.com
```

Comment:
we don't need this one - running lakefs doesn't use it and the name treeverse/lakefs is always docker hub. this is the default. if not you need to specify the address of your repository.

Reply:
How else would I pull images that exist only in ECR?
Remember that when Nessie pulls images, it's not pushed to docker hub yet.